modules/node-tuning-operator.adoc

```diff
@@ -23,13 +23,25 @@ ifdef::operators[]
 [discrete]
 == Purpose
 endif::operators[]
-The Node Tuning Operator helps you manage node-level tuning by orchestrating the TuneD daemon. The majority of high-performance applications require some level of kernel tuning. The Node Tuning Operator provides a unified management interface to users of node-level sysctls and more flexibility to add custom tuning specified by user needs.
+The Node Tuning Operator helps you manage node-level tuning by orchestrating the TuneD daemon and achieves low latency performance by using the Performance Addon controller. The majority of high-performance applications require some level of kernel tuning. The Node Tuning Operator provides a unified management interface to users of node-level sysctls and more flexibility to add custom tuning specified by user needs.
```

Comment:
it's Performance Addon controller or Performance Profile Controller? 

https://github.com/openshift/cluster-node-tuning-operator/pull/373

Reply:
Apart from the above clarification mentioned by @marioferh  rest everything looks good to me.

Reply:
Verified the name: Performance Profile controller. Updated accordingly.